news/2017-07-10-osucatch-world-cup-2017-results.md

```diff
@@ -0,0 +1,33 @@
+
+<center><iframe src="https://player.twitch.tv/?video=v157762013&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe><a href="https://www.twitch.tv/osulive?tt_medium=live_embed&tt_content=text_link" style="padding:2px 0px 4px; display:block; width:345px; font-weight:normal; font-size:10px; text-decoration:underline;">Watch live video from osulive on www.twitch.tv</a></center>
+
+South Korea could claim a tactical advantage from the start, having seen China's performance on nearly every map in the pool after their breathtaking 6-5 victory over Indonesia the previous week. They knew the DoubleTime strength of the Chinese team, again choosing to ban [ChouCho - Yasashisa no Riyuu](https://osu.ppy.sh/b/1144716&m=2). China responded in kind, again banning one of the two AR8 Hidden picks, [Grand Thaw - Aventyr](https://osu.ppy.sh/b/1055601&m=2). China would, unsurprisingly, kick off with DoubleTime, locking in [NeLiME - CODE NAME : ZERO](https://osu.ppy.sh/b/374235&m=2), with captain Dusk finally securing a winning roll and having the chance to pick first. The similarities would end here however, as South Korea pulled out a turnaround victory to break from the start. As expected, a Hidden pick followed, with the South Korean team electing to unsettle China with [Susumu Hirasawa - The Secret of The Flowers of Phenomenon](https://osu.ppy.sh/b/201974&m=2), resulting in another comfortable victory.
+
+China, looking to settle the nerves and secure a point on the scoreboard, would stick with DoubleTime, choosing the consistency-demanding  converted osu! difficulty [3L - Endless night](https://osu.ppy.sh/b/430371&m=2), hoping for Dusk to achieve a repeat double S performance and a comfortable victory. Once again this was scouted by South Korea, switching in Berea who proceeded to drop only a single note and usher in a second break point. A switch to NoMod almost backfired due to a string of early misses on the South Korean pick of [dj TAKA - quaver](https://osu.ppy.sh/b/915210&m=2), but once again though, a slender Chinese advantage would be turned upside-down in the closing moments, with Callionet dropping an entire section, costing himself a would-be spectacular Full Combo, and China their fourth chance at a point.
```

Comment:
double whitespace after ``consistency-demanding``

split paragraph at "a switch to NoMod"